lib/javascript/BasicEvaluatedExpression.js

```diff
@@ -122,6 +129,7 @@ class BasicEvaluatedExpression {
 		if (this.isNull()) return "null";
 		if (this.isString()) return this.string;
 		if (this.isNumber()) return `${this.number}`;
+		if (this.isBigInt()) return `${this.bigint}n`;
```

Comment:
```suggestion
		if (this.isBigInt()) return `${this.bigint}`;
```

This is used for e. g. `${123n}`, which prints "123"